src/Drawable.js

```diff
@@ -433,6 +433,16 @@ class Drawable {
             return true;
         }
 
+        // If the effect bits for mosaic, pixelate, whirl, or fisheye are set, use linear
+        if (this._effectBits & (
```

Comment:
I think you want parens around the statement before !== 0, because !== is taking precedence over &

Reply:
Good catch